### DISCUSSION TITLE
3D model on Android failing with error message

### DISCUSSION THREAD

**Discussion by JacobInCode** [google/model-viewer#1856] [Q&A]
Android users can't view my .glb in AR. When SceneViewer is default they get this error message and when webxr is default they get a black screen. Thanks for help!

![IMG_4777](https://user-images.githubusercontent.com/24348787/105285609-9423cd00-5b69-11eb-8ca9-125d38f31cc4.PNG)

**Comment by elalish** [comment]
First, check the validator: https://github.khronos.org/glTF-Validator/

I'd like to understand the WebXR issue better. What is your device and OS and browser versions? Has everything been updated? What version of model-viewer?
